be2-scala/src/main/scala/ch/epfl/pop/config/RuntimeEnvironment.scala

```diff
@@ -76,4 +54,39 @@ object RuntimeEnvironment {
     addressList
   }
 
+  private def getConfDir: String = {
+    /* Get config directory path from JVM */
+    if (sp("clean") != null) {
+      // starting the program with fresh database
+      println("Starting the server without any previous persistent state")
+
+      // removing database folder
+      val directory = new Directory(new File(dbPath))
+      if (directory.deleteRecursively()) {
+        println("Removed old database folder")
+      }
+    }
+
+    val virtualMachineParam = "scala.config"
```

Comment:
nitpick: as this is an important value (used as a parameter to launch the program), it could be turned into an attribute instead of being inlined in the function for better visibility

Reply:
The same goes for `dbPathParam` and "/database" below, and `sp("clean")` above